examples/checkpoint/toy_model.py

```diff
@@ -31,6 +34,12 @@ def __getitem__(self, idx):
 
 
 class ExampleCode(torch.nn.Module):
+    """
+    A normal pytorch model(nn.Module) with a loss.
+    The loss is calculated in the `forward` function which lets pippy
+    automatically run a .backward(). Pippy handles this backward call
+    because of the nontrivial structure(FillDrain schedule) of the model pipeline.
```

Comment:
Nice comment! This is great!
@kwen2501 Could you double check if this understanding is accurate.